### DISCUSSION TITLE
Importing Bookmarks

### DISCUSSION THREAD

**Discussion by AtreideGET** [karakeep-app/karakeep#386] [Ideas]
Hi,
Is it possible to import links from a csv file?

**Comment by kamtschatka** [comment]
No, but you can simple open it with any tool that can open csv files in a table like manner and copy out the column that contains the URLs and then paste them into hoarder

**Reply by AtreideGET** [reply]
Yes, I imported my favorites with an html file, but I didn't get my lists and tags back. Maybe in a future update.

**Reply by kamtschatka** [reply]
where did you get the html file from? i am not aware that the netscape format we support, actually supports tags

**Reply by AtreideGET** [reply]
I was actually using LinkAce. I exported my links as an html file and as a cvs file (with links/lists/tags). It was this html file that I imported. And it's working.

**Comment by kamtschatka** [comment]
closing, since the import worked with the way described above
